packages/website/docs/tips.md

```diff
@@ -2,129 +2,70 @@
 title: Tips for a good search
 ---
 
-DocSearch can work with almost any website, but we've found that some site
-structures yield more relevant results or faster indexing time. In this page
-we'll share some tips on how you can make the most out of DocSearch.
+DocSearch can work with almost any website, but we've found that some site structures yield more relevant results or faster indexing time. In this page we'll share some tips on how you can make the most out of DocSearch.
```

Comment:
This page seems really similar to the end of `packages/website/docs/required-configuration.mdx` maybe you could gather the content